_config.yml

```diff
@@ -1,5 +1,6 @@
 title: Beaker-Next Design
-theme: "just-the-docs"
+remote_theme: "pmarsceill/just-the-docs"
```

Comment:
Very strange this seems to be a user's name.  It's a little concerning.

Reply:
Actually yes, it is.
Github can serve a small set of Jekyll themes out of the box, but you can also point to a "remote theme" to customize it.
I am pointing to github.com/pmarsceill/just-the-docs, which is the repository of the open source theme.